code/modules/food/recipes_microwave.dm

```diff
@@ -1193,157 +763,540 @@ I said no!
-
-/datum/recipe/microwave/cake/apple
-	fruit = list("apple" = 1)
-	reagents = list("milk" = 5, "flour" = 15, "egg" = 9,"sugar" = 5)
-	result = /obj/item/weapon/reagent_containers/food/snacks/sliceable/applecake
-
-/datum/recipe/microwave/cake/brain
-	reagents = list("milk" = 5, "flour" = 15, "sugar" = 15, "egg" = 9)
-	items = list(/obj/item/organ/internal/brain)
-	result = /obj/item/weapon/reagent_containers/food/snacks/sliceable/braincake
-
-/datum/recipe/microwave/bagelplain
+/datum/recipe/bagelplain
 	reagents = list("water" = 5)
 	items = list(
 		/obj/item/weapon/reagent_containers/food/snacks/bun
 	)
 	result = /obj/item/weapon/reagent_containers/food/snacks/bagelplain
 
-/datum/recipe/microwave/bagelsunflower
+/datum/recipe/bagelsunflower
 	reagents = list("water" = 5)
 	items = list(
 		/obj/item/weapon/reagent_containers/food/snacks/bun,
 		/obj/item/weapon/reagent_containers/food/snacks/rawsunflower
 	)
 	result = /obj/item/weapon/reagent_containers/food/snacks/bagelsunflower
 
-/datum/recipe/microwave/bagelcheese
+/datum/recipe/bagelcheese
 	reagents = list("water" = 5)
 	items = list(
 		/obj/item/weapon/reagent_containers/food/snacks/bun,
 		/obj/item/weapon/reagent_containers/food/snacks/cheesewedge
 	)
 	result = /obj/item/weapon/reagent_containers/food/snacks/bagelcheese
 
-/datum/recipe/microwave/bagelraisin
+/datum/recipe/bagelraisin
 	reagents = list("water" = 5)
 	items = list(
 		/obj/item/weapon/reagent_containers/food/snacks/bun,
 		/obj/item/weapon/reagent_containers/food/snacks/no_raisin
 	)
 	result = /obj/item/weapon/reagent_containers/food/snacks/bagelraisin
 
-/datum/recipe/microwave/bagelpoppy
+/datum/recipe/bagelpoppy
 	fruit = list("poppy" = 1)
 	reagents = list("water" = 5)
 	items = list(
 		/obj/item/weapon/reagent_containers/food/snacks/bun
 	)
 	result = /obj/item/weapon/reagent_containers/food/snacks/bagelpoppy
 
-/datum/recipe/microwave/bageleverything
+/datum/recipe/bageleverything
 	reagents = list("water" = 5)
 	items = list(
 		/obj/item/weapon/reagent_containers/food/snacks/bun,
 		/obj/item/weapon/fuel_assembly/supermatter
 	)
 	result = /obj/item/weapon/reagent_containers/food/snacks/bageleverything
 
-/datum/recipe/microwave/bageltwo
+/datum/recipe/bageltwo
 	reagents = list("water" = 5)
 	items = list(
 		/obj/item/weapon/reagent_containers/food/snacks/bun,
 		/obj/item/device/soulstone
 	)
 	result = /obj/item/weapon/reagent_containers/food/snacks/bageltwo
+	
+/////////////////////////////////////////////////////////////
+//Synnono Meme Foods
+//
+//Most recipes replace reagents with RECIPE_REAGENT_REPLACE
+//to simplify the end product and balance the amount of reagents
+//in some foods. Many require the space spice reagent/condiment
+//to reduce the risk of future recipe conflicts.
+/////////////////////////////////////////////////////////////
+
+
+/datum/recipe/redcurry
+	reagents = list("cream" = 5, "spacespice" = 2, "rice" = 5)
```

Comment:
these all appear to be missing appliances, or else are misplaced in recipes_microwave.dm

Reply:
Doesn't need appliance, datum/recipe assumes MICROWAVE is the default.